File: Pattern/Python/Pattern22.py
n = int(input("Enter the size of the matrix: "))

for i in range(2*n-1):

    for j in range(2*n-1):

        minDist = min(i,j,((2 * n - 2) - i),((2 * n - 2) - j))

        print(n - minDist, end=" ")

    print()

File: Pattern/Java/Pattern22.java
import java.util.*;

public class Pattern22 {

    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);


        System.out.print("Enter the square matrix size: ");
        int n = sc.nextInt();

        for(int i = 0; i < (2 * n - 1); i++){
            
            for(int j = 0; j < (2 * n - 1); j++){

                int minDist = Math.min(Math.min(i, (2 * n - 2) - i), Math.min(j, (2 * n - 2) - j));

                System.out.print(n-minDist + " ");

            }

            System.out.println("");
        }
    }
}